### DISCUSSION TITLE
D1 mini32 no output on output5

### DISCUSSION THREAD

**Discussion by Techdude06** [forkineye/ESPixelStick#792] [General]
Hi,

I have a bunch of espixelsticks v3 and I am converting them over to D1 mini32's. I am having an issue where when testing using the effects rainbow, the D1 outputs the first frame and then doesn't update the pixels after that. They don't seem to output anything when set to FPP remotes. I am having this issue on 4.0 beta 5 through the ci build 10/21. I updated to the latest experimental version (on the web flasher) and I haven't seen that issue yet. Anyone have any ideas what the problem might be?

Thanks.

**Comment by MartinMueller2003** [comment]
Is this still happening? I am not getting any other reports of this. I do use the QuinLed ESP32 processor from the Quad controller and it works after removing the reset pin.
